### ISSUE TITLE
How to expire records utilizing redis timeout?

### ISSUE THREAD

**Issue by brandonsimpson** [balderdashy/sails-redis#50]
Looking through the lib/database/index.js code and I don't see anything referencing setting an expiration or timeout on the records. Is this implemented somehow and I'm just missing it? Seems like a basic foundation of using a cache server, and so far it only looks like this redis adapter is functioning like any other database connection and lacking some redis native functionality.

http://redis.io/commands/expire

If it's not implemented yet, I'd like to contribute. How can I get involved?

**Comment by jonathanwiesel** [comment]
It looks like it was implemented in the 0.9.x branch but for some reason it didn't made it to the master branch, I would love to see this feature, any news on it?

**Comment by Ryanc1256** [comment]
Ill try to move that request into a Pull requests so the other guys can pull it into the master branch.

**Comment by sailsbot** [comment]
Thanks for posting, @brandonsimpson.  I'm a repo bot-- nice to meet you!

It has been 30 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
